third_party/k8s.io/kubernetes/hack/verify-licenses.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
```

Comment:
We need to have the base commit that we are branching from so we can rebase.

Ironically, we also need the LICENSE file

Reply:
The LICENSE file can be found here https://github.com/open-policy-agent/gatekeeper/blob/master/third_party/k8s.io/kubernetes/LICENSE and was added by @sozercan w the cherry pick script.

Per our offline chat, I can add a README file to point to the commit sha we're taking this from as with https://github.com/open-policy-agent/gatekeeper/tree/master/third_party/sigs.k8s.io/controller-runtime

Reply:
Added README